spacy/tokens/span.pyx

```diff
@@ -422,6 +426,48 @@ cdef class Span:
         else:
             raise ValueError(Errors.E030)
 
+    @property
+    def sents(self):
+        """Obtain the sentences that contain this span. If the given span
+        crosses sentence boundaries, return all sentences it is a part of.
+
+        RETURNS (Iterable[Span]): All sentences that the span is a part of.
+
+         DOCS: https://spacy.io/api/span#sents
+        """
+        cdef int start
+        cdef int i
+
+        if not self.doc.has_annotation("SENT_START"):
+            raise ValueError(Errors.E030)
+
+        if "sents" in self.doc.user_span_hooks:
+            yield from self.doc.user_span_hooks["sents"](self)
+        elif "sents" in self.doc.user_hooks:
+            for sentence in self.doc.user_hooks["sents"](self.doc) :
+                if sentence.end > self.start :
+                    if sentence.start < self.end or sentence.start == self.start == self.end:
+                        yield sentence
+                    else :
+                        break
```

Comment:
I believe there is no support for `black` formatting in Cython, so you can't know this, but I'd at least put the `:` directly after the line without space, to match the formatting in other parts of the code base.

Reply:
I updated the formatting